### PR TITLE
docs: add ma option states storybook

### PR DESCRIPTION
**Pull request by Samitier**
Add documentation for ma option states

### CODE REVIEW

File: src/components/MaOption/MaOption.stories.js [hunk 1]
```diff
@@ -29,57 +27,65 @@ export default {
...
+      <ma-option type="${type}" :card="${card}" has-error >
+        Error
+      </ma-option>
+      <ma-option type="${type}" :card="${card}" ref="focusElem" >
+        Focus
+      </ma-option>
+    </ma-stack>
+  `,
+  mounted() {
+    this.$refs.focusElem.$el.focus()
```
Comment (afontcu) [inline src/components/MaOption/MaOption.stories.js:71]: 👌

**Review by afontcu — APPROVED** [review]
LGTM!

I think all vetur diffs will be gone if you run `npm run format`

**Comment by Samitier** [comment]
> I think all vetur diffs will be gone if you run `npm run format`

Nope... still there. Something is happening with my eslint

**Pull request merged**